beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -62,6 +62,11 @@ const ATTESTATION_CACHE_LOCK_TIMEOUT: Duration = Duration::from_secs(1);
 /// validator pubkey cache.
 const VALIDATOR_PUBKEY_CACHE_LOCK_TIMEOUT: Duration = Duration::from_secs(1);
 
+pub const BEACON_CHAIN_DB_KEY: &str = "PERSISTEDBEACONCHAINPERSISTEDBEA";
+pub const OP_POOL_DB_KEY: &str = "OPPOOLOPPOOLOPPOOLOPPOOLOPPOOLOP";
+pub const ETH1_CACHE_DB_KEY: &str = "ETH1CACHEETH1CACHEETH1CACHEETH1C";
+pub const FORK_CHOICE_DB_KEY: &str = "FORKCHOICEFORKCHOICEFORKCHOICEFO";
```

Comment:
Given that there's a `DBColumn` for each of these which provides a unique key prefix, we could use `Hash256::zero` as the key and do away with the bizarre naming convention? We could instantiate a `const` `Hash256` for each of them to keep it clear, or just do it inline (I have a slight pref for multiple consts).

```rust
pub const BEACON_CHAIN_DB_KEY: Hash256 = Hash256([0; 32]);
```

Reply:
NB: `Hash256::zero` won't be `const` until the next ethereum-types release

Reply:
Unfortunately we can do 

```
pub const BEACON_CHAIN_DB_KEY: Hash256 = Hash256([0; 32]);
```

because the `0` element is private :(

Reply:
I got as close as I could, I replaced the strings with `[u8; 32]`. What are your thoughts?

Reply:
Interestingly, the field is pub (as per [the docs](https://docs.rs/ethereum-types/0.7.0/ethereum_types/struct.H256.html)), but the type alias doesn't rename the struct constructor, so you have to do:

```rust
pub const BEACON_CHAIN_DB_KEY: Hash256 = ethereum_types::H256([0; 32]);
```

Or we could put this in `types/src/lib.rs`:

```rust
#[allow(non_snake_case)]
pub const fn Hash256(v: [u8; 32]) -> Hash256 {
    H256(v)
}
```

Reply:
Hmm, seems like we have to either:

- Add `ethereum_types` as a dependency and remove the abstraction provided by `types`
- Make a static function that kind of looks like a constructor that is called a slightly different thing to what it produces (`Hash256` vs `H256`).

I'm a bit uncertain that these are better than what we have now.

Reply:
I quite like the fake constructor, because it makes `Hash256` exactly like `H256`, but I won't block the merge on it. Happy to leave as-is.